openedx/core/djangoapps/course_roles/readme.rst

```diff
@@ -0,0 +1,15 @@
+Course Roles
+#############################
+
+Status: Development
+
+Purpose
+*******
+
+The intention of this app is to to make course level roles for users within LMS/CMS more flexible. 
```

Comment:
Is this app's responsibilities intended to extend to things that are not courses, like Libraries?

Reply:
This app will be responsible for course level roles and permissions. It is not intended (at this time) to extend to other  contexts (v2 libraries, organization level, instance level, etc.)